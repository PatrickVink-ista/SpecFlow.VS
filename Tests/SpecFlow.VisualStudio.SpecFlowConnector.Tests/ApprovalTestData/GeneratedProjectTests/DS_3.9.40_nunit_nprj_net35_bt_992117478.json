{
  "GeneratorOptions": {
    "FeatureFileCount": 1,
    "ScenarioPerFeatureFileCount": 1,
    "CreatedFor": "net3.5",
    "TargetFramework": "v3.5",
    "SpecFlowPackageVersion": "3.9.40"
  }
}
