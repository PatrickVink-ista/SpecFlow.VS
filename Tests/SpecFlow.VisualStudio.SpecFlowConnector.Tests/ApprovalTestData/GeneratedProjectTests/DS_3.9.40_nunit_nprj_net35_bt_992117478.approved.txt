{
  "StepDefinitions": [
    {
      "Type": "Then",
      "Regex": "^lorem dictum (\\d+) accumsan$",
      "Method": "ConubiaDonecEgestasQuisqueSteps.ThenAnteCursusJustoId(Int32)",
      "ParamTypes": "i",
      "Scope": null,
      "Expression": "lorem dictum (\\d+) accumsan",
      "Error": null,
      "SourceLocation": "#0|10|9|12|10"
    },
    {
      "Type": "When",
      "Regex": "^nulla nec dui (\\d+)$",
      "Method": "ConubiaDonecEgestasQuisqueSteps.WhenSemDonecCommodoEt(Int32)",
      "ParamTypes": "i",
      "Scope": null,
      "Expression": "nulla nec dui (\\d+)",
      "Error": null,
      "SourceLocation": "#0|16|9|18|10"
    },
    {
      "Type": "Given",
      "Regex": "^id nulla (\\d+)$",
      "Method": "ConubiaDonecEgestasQuisqueSteps.GivenConubiaDonecDiamEleifend(Int32)",
      "ParamTypes": "i",
      "Scope": null,
      "Expression": "id nulla (\\d+)",
      "Error": null,
      "SourceLocation": "#0|22|9|24|10"
    }
  ],
  "SourceFiles": {
    "0": "<<targetFolder>>\\StepDefinitions\\ConubiaDonecEgestasQuisqueSteps.cs"
  },
  "TypeNames": {}
}
